src/spring-cloud/setup.py

```diff
@@ -16,7 +16,8 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '3.0.1'
+# TODO(jiec): remove this version number after verification.
+VERSION = '3.0.1-use-0301-api'
```

Comment:
Will remove these 2 lines of code, add this now since I'm running CLI E2E test, need a version tag to track which version of code base I'm testing against.